src/Utils/QueryAnalyzer.php

```diff
@@ -106,6 +112,20 @@ public function init(): void {
 			return;
 		}
 
+		// Filter the header key limit. Default is 8000 (8k) to play nice
```

Comment:
Not totally familiar with the expected frontend vernacular, but if we're taking binary Bytes as the input (8192), than maybe this should also say `Default is 8192

Reply:
ah ya, it is 8192, not 8000. 🤦🏻